src/components/organisms/DataExplorer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Pane } from 'evergreen-ui';
+import DataResultItem from '../molecules/DataResultItem';
+import { useQuery } from '@apollo/client';
+import { METADATA, MetadataEntry, Metadata } from '../../queries/metadata';
+import Loading from '../atoms/Loading';
+import ErrorMessage from '../atoms/ErrorMessage';
+
+const DataExplorer: React.FC = () => {
+    const { data, loading, error } = useQuery<Metadata>(METADATA);
+
+    if (loading) {
+        return (
+            <Pane height="100vh">
```

Comment:
Is this necessary? What if there's something above it?

Reply:
I thought it was to get the loading icon centred because i was using storybook.
It should probably be changed to just 100%.